frontend/public/components/start-guide.tsx

```diff
@@ -70,9 +68,7 @@ export const withStartGuide = (WrappedComponent, disable: boolean = true) =>
               <OpenShiftGettingStarted canCreateProject={flags[FLAGS.CAN_CREATE_PROJECT]} />
             </HintBlock>
           </div>
-          {// Whitelist certain resource kinds that should not be disabled when no projects are
-          // available. Disabling should also be optional
-          !disable || WHITELIST.includes(kind) ? (
+          {!disable || !rest?.kindObj?.namespaced ? (
```

Comment:
We only want to test `!namespaced` if `kindObj` is defined. Looking above, I believe it's possible to pass a `kind` as a prop instead of a `kindObj`. So I'd change the logic to

```suggestion
          {!disable || (rest.kindObj && !rest.kindObj.namespaced) ? (
```

With the current change if `kindObj` was undefined, `!rest?.kindObj?.namespaced` would evaluate to `true`, not showing the disabled page.